src/converter/NamespaceResolver.kt

```diff
@@ -0,0 +1,18 @@
+package converter
+
+class NamespaceResolver(val aliases: Map<String, String>) {
```

Comment:
How about imports from external namespaces? Maybe using tsc resolve would cover all required cases.

Reply:
Support for imports of external namespaces is out of scope of this PR